src/content/configuration/output.md

```diff
@@ -330,6 +330,10 @@ The following substitutions are available in template strings (via webpack's int
 
 The lengths of `[hash]` and `[chunkhash]` can be specified using `[hash:16]` (defaults to 20). Alternatively, specify [`output.hashDigestLength`](#outputhashdigestlength) to configure the length globally.
 
+In places where substitutions are supported but you don't want them to be substituted, for example when wanting to output a bundle with the literal name `[name].js`, you can escape it by adding backslashes between the brackets.
+
+Example: `[\id\]` generates `[id]` instead of getting replaced with the `id`.
```

Comment:
`\[id\]`

Reply:
The escaping is with the backslashes before and after the variable name inside the brackets so it should be `[\id\]`